packages/block-library/src/query/editor.scss

```diff
@@ -49,6 +49,12 @@
 	}
 }
 
+.block-library-query-toolspanel__design {
+	.block-library-query-pattern__selection-content {
+		margin-top: $grid-unit-10;
```

Comment:
Is this an override of sorts?

Reply:
It's to compensate for the removal of the search component in the patterns view.